### DISCUSSION TITLE
Welcome to SoqlX Discussions!

### DISCUSSION THREAD

**Discussion by superfell** [superfell/SoqlX#133] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by VishwamberShetty** [comment]
Hello everyone,

I recently discovered your fantastic tool and can't believe I didn't know about it sooner. Nonetheless, I am thrilled to have found it and had the opportunity to use it.

I have a few requests for new features that I will be submitting over the next few days. I hope you will consider them.

While I cannot contribute to the tool's development, I am more than willing to assist with testing and providing Salesforce-related help if needed.

Thank you,
Vishwamber Shetty
